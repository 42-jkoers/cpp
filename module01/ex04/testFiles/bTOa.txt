bb a bb aa b a
