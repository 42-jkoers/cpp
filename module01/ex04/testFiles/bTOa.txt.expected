aa a aa aa b a
